ui/packages/consul-ui/tests/acceptance/dc/nspaces/delete.feature

```diff
@@ -45,7 +46,7 @@ Feature: dc / nspaces / delete: Deleting items with confirmations, success and e
     Given the url "[URL]" responds with a 500 status
     And I click delete
     And I click confirmDelete
-    And "[data-notification]" has the "notification-delete" class
+    And "[data-notification]" has the "notification-update" class
     And "[data-notification]" has the "error" class
```

Comment:
New approach doesn't have a separate class for deletions. Having these separate classes are a bit of a hangover from previous things. The other changes here are just hardcoding in the nspace instead of using a table with a single row to do the test (if you look at the entire file you'll see what I mean)